### PR TITLE
refactoring

### PR DESCRIPTION
**Pull request by kevmoo**
@jacob314 Care to look at this, too?

### CODE REVIEW

File: lib/infohelper.dart [hunk 1]
```diff
@@ -2,6 +2,8 @@
 // for details. All rights reserved. Use of this source code is governed by a
 // BSD-style license that can be found in the LICENSE file.
 
+library dump_viz.info_helper;
```
Comment (jacob314) [inline lib/infohelper.dart]: change file to 
info_helper.dart to match library name.

File: lib/dragdrop.dart [hunk 2]
```diff
@@ -2,7 +2,10 @@
 // for details. All rights reserved. Use of this source code is governed by a
 // BSD-style license that can be found in the LICENSE file.
 
-part of viewer;
+library dump_viz.dragdrop;
```
Comment (jacob314) [inline lib/dragdrop.dart]: not clear why this is dragdrop
instead of
drag_drop

**Comment by kevmoo** [comment]
@jacob314 @jakemac53 PTAL - A bunch more cleanup here

File: lib/src/drap_drop_file.dart [hunk 3]
```diff
@@ -47,17 +34,33 @@ class DragDropFile {
...
+
+  void _loadFile(File file) {
+    _selectedFile = file;
+    document.title = file.name + " - Dump Info Viewer";
+    FileReader reader = new FileReader();
+    reader.onLoad.listen((e) {
+      String fileContents = reader.result;
+      // Substring because fileContents contains the mime type
+      var contents =
+      window.atob(fileContents.substring(fileContents.indexOf(',') + 1));
```
Comment (jakemac53) [inline lib/src/drap_drop_file.dart]: formatter

File: lib/src/logical_row.dart [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+      }
+      this.children.forEach((child) => child.show(before: this.rowElement));
+    } else {
+      this.children.forEach((child) => child.hide());
+    }
+    this.rowElement.setArrow(this.children.isNotEmpty, open);
+  }
+
+  void hide() {
+    //this.parentElement.children.remove(this.getElement());
```
Comment (jakemac53) [inline lib/src/logical_row.dart]: remove?

**Comment by jakemac53** [comment]
lgtm

File: lib/src/logical_row.dart (hunk 4 above)
Comment (kevmoo) [inline lib/src/logical_row.dart]: done

**Pull request merged**